packages/react-scripts/scripts/build.js

```diff
@@ -112,7 +112,7 @@ function build(previousFileSizes) {
       if (messages.errors.length) {
         return reject(new Error(messages.errors.join('\n\n')));
       }
-      if (process.env.CI && messages.warnings.length) {
+      if (process.env.CI && process.env.CI.toLowerCase() === 'true' && messages.warnings.length) {
```

Comment:
I have a feeling people might set this to things like `travis`, etc.

How about checking `!== 'false'`?